integration/helpers/utils.ts

```diff
@@ -4,7 +4,7 @@ import { exec } from "child_process";
 
 const AppPaths: Partial<Record<NodeJS.Platform, string>> = {
   "win32": "./dist/win-unpacked/Lens.exe",
-  "linux": "./dist/linux-unpacked/kontena-lens",
+  "linux": "./dist/linux-unpacked/lens",
```

Comment:
does this have any side effect?

Reply:
It might cause a problem with `snap` but I cannot be sure before we do a pre-release.